src/eval-frame/components/reps/tiny-reps.jsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import styled from "react-emotion";
+
+const numberColor = "rgb(28, 0, 207)";
```

Comment:
would `reps-theme.js` in this folder be ok? this whole section feels like it may get carved into a stand-alone module at some point

Reply:
That sounds good to me. Truth be told, I can see down-the-line having one consistent theme for editor + reps, but that's ... pretty far away :)

Reply:
upon reflection, i think that it may be most appropriate to keep these in this file for now until it becomes clearer what the future around this is. ATM this is a little snowflake-y b/c this theme was actually chosen to match 'react-inspector', and since i agree that having one set of colors in the editor and reps is the best long term plan, i'm not sure it makes sense to mix this temporary snowflake with the rest of the theme, nor to break it out since nothing else will use it in the immediate future.